src/ign.hh

```diff
@@ -30,6 +30,7 @@ extern "C" GZ_MSGS_VISIBLE void cmdMsgList();
 
 /// \brief External hook to read the library version.
 /// \return C-string representing the version. Ex.: 0.1.2
-extern "C" GZ_MSGS_VISIBLE const char *ignitionMsgsVersion();
+extern "C" GZ_MSGS_VISIBLE const char *gzMsgsVersion();
+extern "C" GZ_MSGS_VISIBLE const char GZ_DEPRECATED(9) *ignitionMsgsVersion();
```

Comment:
Oh this header isn't installed (it isn't in the `include` directory), so we can do a hard tock

Reply:
[d67ca90](https://github.com/gazebosim/gz-msgs/pull/260/commits/d67ca90169437fe87af6876f6aa60d856596281c)